api-2.7/src/main/java/org/openmrs/module/initializer/api/conceptreferencerange/ConceptReferenceRangeLoader.java

```diff
@@ -0,0 +1,20 @@
+package org.openmrs.module.initializer.api.conceptreferencerange;
+
+import org.openmrs.ConceptReferenceRange;
+import org.openmrs.annotation.OpenmrsProfile;
+import org.openmrs.module.initializer.api.loaders.BaseCsvLoader;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+
+@OpenmrsProfile(openmrsPlatformVersion = "2.7.x")
```

Comment:
This doesn't work. Also, add some future-proofing.

```suggestion
@OpenmrsProfile(openmrsPlatformVersion = "2.7.0 - 2.*")
```